examples/authsignal-passwordless/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
```

Comment:
Please remove any dependencies/files/scripts related to eslint as outlined in https://github.com/vercel/next.js/blob/canary/contributing.md#adding-examples to keep the example focused. 🙏